TestAssets/DesktopTestProjects/DesktopAppWhichCallsDotnet/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*"
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-rc2-*"
   },
```

Comment:
I'm going to step through the other nugget packages and make sure everything is right too